keys.go

```diff
@@ -302,6 +362,46 @@ type ForceOpt struct {
 	Force bool
 }
 
+//ListKeyVersionsOptions struct to add the query parameters for the GetKeyVersions function
+type ListKeyVersionsOptions struct {
+	Limit      *uint64
+	Offset     *uint64
+	TotalCount *bool
+}
+
+// GetKeyVersion gets all the versions of the key resource by specifying ID of the key and/or optional parameteres
+func (c *Client) GetKeyVersions(ctx context.Context, id string, listKeyVersionsOptions *ListKeyVersionsOptions) (*KeyVersions, int, error) {
```

Comment:
let's make this as ListKeyVersions to keep the naming consistent with KP API docs